lib/path.js

```diff
@@ -685,22 +700,22 @@ const win32 = {
 
     if (resolvedPath.length >= 3) {
       var code = resolvedPath.charCodeAt(0);
-      if (code === 92/*\*/) {
+      if (code === CHAR_BACKWARD_SLASH) {
         // Possible UNC root
 
-        if (resolvedPath.charCodeAt(1) === 92/*\*/) {
+        if (resolvedPath.charCodeAt(1) === CHAR_BACKWARD_SLASH) {
           code = resolvedPath.charCodeAt(2);
-          if (code !== 63/*?*/ && code !== 46/*.*/) {
+          if (code !== CHAR_QUESTION_MARK && code !== CHAR_DOT) {
             // Matched non-long UNC root, convert the path to a long UNC path
             return '\\\\?\\UNC\\' + resolvedPath.slice(2);
           }
         }
-      } else if ((code >= 65/*A*/ && code <= 90/*Z*/) ||
-                 (code >= 97/*a*/ && code <= 122/*z*/)) {
+      } else if ((code >= CHAR_UPPERCASE_A && code <= CHAR_UPPERCASE_Z) ||
+                 (code >= CHAR_LOWERCASE_A && code <= CHAR_UPPERCASE_Z)) {
```

Comment:
Typo here. Second line of the `else if` statement, second variable is supposed to be `CHAR_LOWERCASE_Z`.

Reply:
@apapirovski fixed

Reply:
Oh, I did not see that you already commented @apapirovski :D